cmd/kogito/command/shared/install_operator.go

```diff
@@ -55,13 +55,13 @@ var (
 
 // SilentlyInstallOperatorIfNotExists attempts to install the operator and does not log a message if it is installed
 func SilentlyInstallOperatorIfNotExists(namespace string, operatorImage string, client *client.Client) (installed bool, err error) {
-	return InstallOperatorIfNotExists(namespace, operatorImage, client, false, false)
+	return InstallOperatorIfNotExists(namespace, operatorImage, client, false, false, AlphaChannel)
```

Comment:
Do we expect that always the alpha channel will be silently installed? shouldn't we set a default value and use that instead hardcode it?

Reply:
I had created a function getDefaultChannel() under kogitochannel.go file and called it where required

Reply:
I didn't change the behavior, still hardcoded, if user select dev-preview channel the silent install should use it.

Reply:
I had already made those changes. Please refersh your browser window

Reply:
not really, GetDefaultChannel is returning the AlphaCHannel, still hardcoded.

Reply:
@spolti 
With current implementation Kogito_operator can be installed through 10 different endpoints. If we divide them under silent and non silent modes, these are as following : 
**Non Silent mode**
1. kogito install operator
**Silent mode**
2. kogito install data-index
3. kogito install mgmt-console
4. kogito install jobs-service
5. kogito new-project
6. kogito use-project
7. kogito deploy-service
8. kogito install infinispan
9. kogito install kafka
10. kogito install keyclock

As I understood from silent mode, In this mode user is not directly intended to install kogito-operator. For example User want to install data-index service but internally kogito-operator is pre-requisit to install data-index service so our CLI checks whether kogito-operator is already installed or not and if it is not installed then it will first install operator silently(without any user input) and then install data-index service.

Currently we are only taking user choice for channel while installing operator in non-silent mode(i.e, kogito install operator). In silent mode we don't capture user choosen channel and by default install operator using _alpha_ channel.

**_With your comment do you want me to capture channel type in non-silent mode as well and use that value for installing operator in silent mode?_**

Reply:
As discussed, I had added user option to enter channel value in both silent and non-silent mode